src/service/index/xtext.index.service.ts

```diff
@@ -17,4 +17,7 @@ export class XtextIndexService extends IndexService {
     return this.http.post<IndexDelta[]>(this.serviceUrl + '/refresh', null).toPromise().catch(reject => []);
   }
 
+  reload(): Promise<any> {
```

Comment:
maybe add a short comment wrt what reload does in contrast to refresh…? (Or possibly more descriptive names)